packages/create-playwright/src/utils.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import fs from 'fs';
+import { execSync } from 'child_process';
+import path from 'path';
+
+import { prompt } from 'enquirer';
+import colors from 'ansi-colors';
+
+
+export function executeCommands(cwd: string, commands: string[]) {
+  for (const command of commands) {
+    console.log('Running:', command);
+    execSync(command, {
+      stdio: 'inherit',
+      cwd,
+    });
+  }
+}
+
+export async function createFiles(rootDir: string, files: Map<string, string>, force: boolean = false) {
+  for (const [relativeFilePath, value] of files) {
+    const absoluteFilePath = path.join(rootDir, relativeFilePath);
+    if (fs.existsSync(absoluteFilePath) && !force) {
+      const { override } = await prompt<{ override: boolean }>({
+        type: 'confirm',
+        name: 'override',
+        message: `${absoluteFilePath} as it already exists. Should it override?`,
```

Comment:
This sounds strange to me. How does this look like for the user?

Reply:
![image](https://user-images.githubusercontent.com/17984549/133148791-53e3659f-a71a-47d2-a92a-4e2bb5a229cc.png)

Joel and me thought it might be good to have.